ietf/meeting/tests_views.py

```diff
@@ -6835,6 +6854,8 @@ def test_submit_and_approve_multiple_versions(self, mock_slides_manager_cls):
         self.assertIn('third version', contents)
 
 
+
+
```

Comment:
```suggestion
```

Reply:
Not sure where those blank lines came from!